app/uk/gov/hmrc/minorentityidentificationfrontend/connectors/RetrieveTrustKnownFactsConnector.scala

```diff
@@ -28,12 +29,11 @@ import javax.inject.{Inject, Singleton}
 import scala.concurrent.{ExecutionContext, Future}
 
 @Singleton
-class RetrieveTrustKnownFactsConnector @Inject()(httpClient: HttpClient, appConfig: AppConfig)(implicit ec: ExecutionContext) {
+class RetrieveTrustKnownFactsConnector @Inject()(httpClient: HttpClientV2, appConfig: AppConfig)(implicit ec: ExecutionContext) {
 
   def retrieveTrustKnownFacts(sautr: String)(implicit hc: HeaderCarrier): Future[Option[TrustKnownFacts]] = {
-    httpClient.GET(appConfig.retrieveTrustsKnownFactsUrl(sautr))(
+    httpClient.get(url"${appConfig.retrieveTrustsKnownFactsUrl(sautr)}").execute(
```

Comment:
In most of the connectors we define the type after "execute". So it would be execute[TrustKnownFacts]....